projects/packages/codesniffer/changelog/update-php-requirements

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: changed
+
+General: updated PHP requirement to PHP 7.0+
```

Comment:
I'd have done something more like this
```suggestion
BREAKING: Drop support for PHP 5.6. `testVersion` should be set to `7.0-`.
```

Reply:
Done in 6caa55c72efd1f4cd8352fb39b30055590d6efbc